pkg/kubeadm/cmd/init.go

```diff
@@ -67,15 +67,31 @@ func NewCmdInit(out io.Writer, s *kubeadmapi.KubeadmConfig) *cobra.Command {
 
 	cmd.PersistentFlags().IPNetVar(
 		&s.InitFlags.Services.CIDR, "service-cidr", *kubeadmapi.DefaultServicesCIDR,
-		`(optional) use alterantive range of IP address for service VIPs, e.g. "10.16.0.0/12"`,
+		`(optional) use alternative range of IP address for service VIPs, e.g. "10.16.0.0/12"`,
 	)
 	cmd.PersistentFlags().StringVar(
 		&s.InitFlags.Services.DNSDomain, "service-dns-domain", kubeadmapi.DefaultServiceDNSDomain,
-		`(optional) use alterantive domain name for services, e.g. "myorg.internal"`,
+		`(optional) use alternative domain name for services, e.g. "myorg.internal"`,
 	)
 	cmd.PersistentFlags().StringVar(
 		&s.InitFlags.CloudProvider, "cloud-provider", "",
-		`(optional) enable cloud proiver features (external load-balancers, storage, etc)`,
+		`(optional) enable cloud provider features (external load-balancers, storage, etc)`,
+	)
+	cmd.PersistentFlags().StringSliceVar(
```

Comment:
Actually, I don't think this is helpful at all. While it makes a shorter `--help` output, I never found any binary in Unix or Kubernetes that would show more info with a different command, i.e. `--advanced-help`.

Also, is there such a flag `--advanced-help` in `spf13/cobra`?